appserver/concurrent/concurrent-connector/src/main/java/org/glassfish/concurrent/config/ManagedExecutorServiceBase.java

```diff
@@ -92,6 +92,23 @@ public interface ManagedExecutorServiceBase extends ConfigBeanProxy,
      *              {@link String }
      */
     void setLongRunningTasks(String value) throws PropertyVetoException;
+
+    /**
+     * Gets the value of the useForkJoinPool property.
+     *
+     * @return possible object is
+     *         {@link String }
+     */
+    @Attribute(defaultValue="false", dataType=Boolean.class)
+    String getUseForkJoinPool();
+
+    /**
+     * Sets the value of the useForkJoinPool property.
+     *
+     * @param value allowed object is
+     *              {@link String }
+     */
+    void setUseForkJoinPool(String value) throws PropertyVetoException;
```

Comment:
Would this not make more sense to exist under the extending `ManagedExecutorService` interface?